appveyor.yml

```diff
@@ -1,105 +0,0 @@
-environment:
```

Comment:
Again, we can keep this for now until/unless we have a replacement. This will run tests on every PR commit, whereas the wheel build script currently only runs on specific tags.

certainly fine to remove deployment steps from here, though.

Reply:
will have to keep `appveyor/setup_fftw_dlls.cmd` and `appveyor/run_with_env.cmd` files as well, but it does seem safe to delete the `deploy_script` section and `deploy.ps1`.

Reply:
N/A after GHA